neurodsp/utils/data.py

```diff
@@ -110,6 +110,35 @@ def compute_nseconds(sig, fs):
     return len(sig) / fs
 
 
+def compute_cycle_nseconds(freq, fs=None):
+    """Compute the length, in seconds, for a single cycle at a particular frequency.
+
+    Parameters
+    ----------
+    fs :  float
+        Sampling rate, in Hz.
+    freq : float, optional
```

Comment:
These parameters are mixed up. freq is positional and fs is optional.

Reply:
Ooops! Good catch - fixed!